projects/components/src/back-button/back-button.component.ts

```diff
@@ -0,0 +1,22 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { IconType } from '@hypertrace/assets-library';
+import { NavigationService } from '@hypertrace/common';
+import { IconSize } from '../icon/icon-size';
+
+@Component({
+  selector: 'ht-back-button',
+  template: `
+    <div (click)="this.onClickBack()" class="back">
+      <ht-icon icon="${IconType.ArrowLeft}" size="${IconSize.Small}" class="arrow"></ht-icon>
```

Comment:
It says button but we are using just an icon

Reply:
Oh! I got your point. Updated to use `ht-button` component.